debezium-server/debezium-server-redis/src/main/java/io/debezium/server/redis/RedisStreamChangeConsumer.java

```diff
@@ -125,6 +134,9 @@ else if (MESSAGE_FORMAT_COMPACT.equals(messageFormat)) {
                     String.format("Property %s expects value one of '%s' or '%s'", PROP_MESSAGE_FORMAT, MESSAGE_FORMAT_EXTENDED, MESSAGE_FORMAT_COMPACT));
         }
 
+        int memoryThreshold = config.getOptionalValue(PROP_MEMORY_THRESHOLD, Integer.class).orElse(85);
```

Comment:
Add a validation that the property is in the range of 0 and 100. 0 stands for `no check`, right?